pyvista/plotting/tools.py

```diff
@@ -102,163 +113,13 @@ def system_supports_plotting():
-    vtk.vtkAxesActor
-        Axes actor.
-
-    Examples
-    --------
-    Create the default axes marker.
-
-    >>> import pyvista as pv
-    >>> marker = pv.create_axes_marker()
-    >>> pl = pv.Plotter()
-    >>> _ = pl.add_actor(marker)
-    >>> pl.show()
-
-    Create an axes marker at the origin with custom colors and axis labels.
-
-    >>> import pyvista as pv
-    >>> marker = pv.create_axes_marker(
-    ...     line_width=4,
-    ...     ambient=0.0,
-    ...     x_color="#378df0",
-    ...     y_color="#ab2e5d",
-    ...     z_color="#f7fb9a",
-    ...     xlabel="X Axis",
-    ...     ylabel="Y Axis",
-    ...     zlabel="Z Axis",
-    ...     label_size=(0.1, 0.1),
-    ... )
-    >>> pl = pv.Plotter()
-    >>> _ = pl.add_actor(marker)
-    >>> pl.show()
-
-    """
-    x_color = Color(x_color, default_color=pyvista.global_theme.axes.x_color)
-    y_color = Color(y_color, default_color=pyvista.global_theme.axes.y_color)
-    z_color = Color(z_color, default_color=pyvista.global_theme.axes.z_color)
-    axes_actor = _vtk.vtkAxesActor()
-    axes_actor.GetXAxisShaftProperty().SetColor(x_color.float_rgb)
-    axes_actor.GetXAxisTipProperty().SetColor(x_color.float_rgb)
-    axes_actor.GetYAxisShaftProperty().SetColor(y_color.float_rgb)
-    axes_actor.GetYAxisTipProperty().SetColor(y_color.float_rgb)
-    axes_actor.GetZAxisShaftProperty().SetColor(z_color.float_rgb)
-    axes_actor.GetZAxisTipProperty().SetColor(z_color.float_rgb)
-    # Set labels
-    axes_actor.SetXAxisLabelText(xlabel)
-    axes_actor.SetYAxisLabelText(ylabel)
-    axes_actor.SetZAxisLabelText(zlabel)
-    if labels_off:
-        axes_actor.AxisLabelsOff()
-    # Set Line width
-    axes_actor.GetXAxisShaftProperty().SetLineWidth(line_width)
-    axes_actor.GetYAxisShaftProperty().SetLineWidth(line_width)
-    axes_actor.GetZAxisShaftProperty().SetLineWidth(line_width)
-
-    axes_actor.SetConeRadius(cone_radius)
-    axes_actor.SetNormalizedShaftLength([shaft_length] * 3)
-    axes_actor.SetNormalizedTipLength([tip_length] * 3)
-    axes_actor.GetXAxisShaftProperty().SetAmbient(ambient)
-    axes_actor.GetYAxisShaftProperty().SetAmbient(ambient)
-    axes_actor.GetZAxisShaftProperty().SetAmbient(ambient)
-    axes_actor.GetXAxisTipProperty().SetAmbient(ambient)
-    axes_actor.GetYAxisTipProperty().SetAmbient(ambient)
-    axes_actor.GetZAxisTipProperty().SetAmbient(ambient)
-
-    for label_actor in [
-        axes_actor.GetXAxisCaptionActor2D(),
-        axes_actor.GetYAxisCaptionActor2D(),
-        axes_actor.GetZAxisCaptionActor2D(),
-    ]:
-        label_actor.SetWidth(label_size[0])
-        label_actor.SetHeight(label_size[1])
-
-    _update_axes_label_color(axes_actor, label_color)
-
-    return axes_actor
+@wraps(AxesActor.__init__)
+def create_axes_marker(*args, **kwargs):  # numpydoc ignore=RT01,PR01
+    """Wrap ``AxesActor`` constructor."""
+    return AxesActor(*args, **kwargs)
 
 
 def create_axes_orientation_box(
-    line_width=1,
```

Comment:
Please gracefully handle keyword argument deprecations

Reply:
Oh, hm... That's not good. We should really have a lint check to make sure there aren't unused keyword arguments. Thanks for catching that.